pkg/controller/seed-controller-manager/backup/resources.go

```diff
@@ -14,22 +14,23 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package apiserver
+package backup
 
 import (
-	"crypto/x509"
-
 	"k8c.io/kubermatic/v2/pkg/resources"
-	"k8c.io/kubermatic/v2/pkg/resources/certificates"
 	"k8c.io/kubermatic/v2/pkg/resources/reconciling"
+
+	corev1 "k8s.io/api/core/v1"
 )
 
-// DexCACertificateCreator returns a function to create/update the secret with the certificate for TLS verification against dex
-func DexCACertificateCreator(getDexCA func() ([]*x509.Certificate, error)) reconciling.NamedSecretCreatorGetter {
-	return func() (string, reconciling.SecretCreator) {
-		return resources.DexCASecretName, certificates.GetDexCACreator(
-			resources.DexCAFileName,
-			getDexCA,
-		)
+func caBundleConfigMapCreator(name string, caBundle string) reconciling.NamedConfigMapCreatorGetter {
```

Comment:
This function already defined in `pkg/controller/user-cluster-controller-manager/resources/resources/ca-bundle.ConfigMapCreator()`.